Riot/Modules/Room/Views/InputToolbar/RoomInputToolbarView.m

```diff
@@ -80,10 +80,6 @@ - (void)awakeFromNib
     [self updateUIWithAttributedTextMessage:nil animated:NO];
     
     self.textView.toolbarDelegate = self;
-    
-    // Add an accessory view to the text view in order to retrieve keyboard view.
-    inputAccessoryView = [[UIView alloc] initWithFrame:CGRectZero];
-    self.textView.inputAccessoryView = inputAccessoryView;
```

Comment:
Have you checked against https://github.com/vector-im/element-ios/issues/5042 to see if this change causes a regression?

Reply:
Yeah it caused, i've reverted that (including a name change to avoid confusion). I've fixed the issue 3 by updating the constraints of table view in the room screen.